test/realtime/vehicle_or_ghost_test.exs

```diff
@@ -62,38 +66,50 @@ defmodule Realtime.VehicleOrGhostTest do
 
   describe "find_by/2" do
     test "matches on any of run, vehicle, or operator" do
```

Comment:
suggestion (non-blocking): The test was already written this way so no need to update, but maybe it would be helpful to split this into three tests - run, vehicle, and operator since there are a lot of assertions here